### PR TITLE
Document sensitive Adyen data & add information about logs

### PR DESCRIPTION
**Pull request by lkostrowski**

### CODE REVIEW

**Review by witoszekdev — APPROVED** [review]

File: docs/developer/app-store/apps/adyen/overview.mdx [hunk 1]
```diff
@@ -266,3 +266,34 @@ mutation TransactionProcess($transactionId: ID!, $data: JSON, $customerIpAddress
   }
 }
 ```
+
+## Data processing
+
+The app *stores* and *processes* data that may be PIIs or sensitive:
```
Comment (patrys) [inline docs/developer/app-store/apps/adyen/overview.mdx:272]: Correct me if I'm wrong, but while the app processes PII, it does not store PII.

Comment (aniav) [inline docs/developer/app-store/apps/adyen/overview.mdx:272]: Would be good to maybe not combine these two together. We Process (or rather send) PII, but we do not store it. In terms of privacy this is a big deal, so would be cool to make the differentiation in this sentence, just not to scare anyone.

Comment (krzysztofzuraw) [reply docs/developer/app-store/apps/adyen/overview.mdx]: Updated in [`87d749e` (#1415)](https://github.com/saleor/saleor-docs/pull/1415/commits/87d749ec4b8c74491d881efe4c1bbe1d7fc601e5)

Comment (krzysztofzuraw) [reply docs/developer/app-store/apps/adyen/overview.mdx]: Updated in [`87d749e` (#1415)](https://github.com/saleor/saleor-docs/pull/1415/commits/87d749ec4b8c74491d881efe4c1bbe1d7fc601e5)

**Review by witoszekdev — APPROVED** [review]

File: docs/developer/app-store/apps/adyen/overview.mdx [hunk 2]
```diff
@@ -266,3 +266,35 @@ mutation TransactionProcess($transactionId: ID!, $data: JSON, $customerIpAddress
   }
 }
 ```
+
+## Data processing
+
+The app *processes* data that may be PIIs or sensitive:
```
Comment (patrys) [inline docs/developer/app-store/apps/adyen/overview.mdx:272]:
```suggestion
The app transmits and processes PII and sensitive data but does not store any information it processes.
```

File: docs/developer/app-store/apps/adyen/overview.mdx [hunk 3]
```diff
@@ -266,3 +266,35 @@ mutation TransactionProcess($transactionId: ID!, $data: JSON, $customerIpAddress
...
+
+### Processing
```
Comment (patrys) [inline docs/developer/app-store/apps/adyen/overview.mdx:274]: I'd get rid of this header. It repeats what the header directly above it says, and it's the only section here.

File: docs/developer/app-store/apps/adyen/overview.mdx [hunk 4]
```diff
@@ -266,3 +266,35 @@ mutation TransactionProcess($transactionId: ID!, $data: JSON, $customerIpAddress
...
+- The IP address of the customer
+- Browser data that Adyen drop-in is gathering (app is receiving it from the Storefront and sending it to Adyen). See [Adyen docs](https://docs.adyen.com/online-payments/analytics-and-data-tracking/)
+- Saleor customer ID
+- Checkout / order language
+- Line items from checkout / order (SKUs, descriptions, etc)
+- Customer email
+- Customer name, surname
+- Customer phone
+- Customer billing and shipping address
+
```
Comment (patrys) [inline docs/developer/app-store/apps/adyen/overview.mdx:287]: Suggestion:

The app does not process any Cardholder Data (CHD) or Sensitive Authentication Data (SAD), as defined by the PCI DSS regulations. Such information is exchanged directly between the Adyen storefront components and the Adyen servers.

**Review by patrys — CHANGES_REQUESTED** [review]

**Pull request merged**